src/codemagic/models/provisioning_profile.py

```diff
@@ -79,7 +79,8 @@ def team_name(self) -> str:
 
     @property
     def has_beta_entitlements(self) -> bool:
-        return self._plist.get('Entitlements', dict()).get('beta-reports-active', False)
+        entitlements = self._plist.get('Entitlements', dict())
+        return next((entitlements[key] for key in entitlements.keys() if 'beta-reports-active' in key), False)
```

Comment:
This would be easier to read:
```suggestion
        for key, value in entitlements.items():
            if 'beta-reports-active' in key:
                return value
        return False
```

Reply:
fixed in d7de32a87dd4bfd264c78c390a182d62a95526cd